src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -340,6 +368,54 @@ public void setCashAdvancePaymentCodeName(String cashAdvancePaymentCodeName) {
         this.cashAdvancePaymentCodeName = cashAdvancePaymentCodeName;
     }
 
+    public KualiDecimal getCashAdvanceAmount() {
+        return cashAdvanceAmount;
+    }
+
+    public void setCashAdvanceAmount(KualiDecimal cashAdvanceAmount) {
+        this.cashAdvanceAmount = cashAdvanceAmount;
+    }
+
+    public String getCashAdvanceTransactionType() {
+        return cashAdvanceTransactionType;
+    }
+
+    public void setCashAdvanceTransactionType(String cashAdvanceTransactionType) {
+        this.cashAdvanceTransactionType = cashAdvanceTransactionType;
+    }
+
+    public ConcurSaeRequestedCashAdvanceDetailLineValidationResult getValidationResult() {
+        return validationResult;
+    }
+
+    public void setValidationResult(ConcurSaeRequestedCashAdvanceDetailLineValidationResult validationResult) {
+        this.validationResult = validationResult;
+    }
+
+    public String getEmployeeChart() {
+        return employeeChart;
+    }
+
+    public void setEmployeeChart(String employeeChart) {
+        this.employeeChart = employeeChart;
+    }
+
+    public String getEmployeeAccountNumber() {
+        return employeeAccountNumber;
+    }
+
+    public void setEmployeeAccountNumber(String employeeAccountNumber) {
+        this.employeeAccountNumber = employeeAccountNumber;
+    }
+
+    public String getCashAdvanceName() {
+        return cashAdvanceName;
+    }
+
+    public void setCashAdvanceName(String cashAdvanceName) {
+        this.cashAdvanceName = cashAdvanceName;
+    }
+
     public String toString() {
         StringBuilder sb = new StringBuilder(" batchID: ").append(batchID).append(" batchDate: ").append(batchDate);
```

Comment:
This whole function could be cleaned up by using ToStringBuilder, check out AccountingXmlDocumentAccountingLine for how to use it.

Reply:
Since the purpose of this user story is to do the minimal number of changes to the existing SAE processing and this is a fundamental class of that processing with just the needed new data elements being added, I would prefer to have this cleanup performed in a future user story when the request extract processing is removed.  Are you amenable to that?

Reply:
Yes, that is fine